storage.js

```diff
@@ -0,0 +1,34 @@
+const userName = document.getElementById('user-name');
+const userEmail = document.getElementById('user-email');
+const userText = document.getElementById('user-text');
+const contactForm = document.querySelector('form');
+const resetBtn = document.querySelector('.form-reset');
```

Comment:
- [ ] The issue that is in the code is because `UserName` and `userText` are null. that is because they are pointed to elements that doesn't exit in your page(index.html). `document.getElementById('user-name');` or `document.getElementById('user-text');` they don't exist because they are no elements with those IDs.  so to fix it, make sure those elements have those corresponding IDs.

![Screenshot(9)](https://user-images.githubusercontent.com/45217364/175767551-4b9a37e5-7daa-4fa2-b664-84437e8ef77c.png)